### PR TITLE
{,x-pack}/packetbeat: add support for pipeline module uploads

### PR DESCRIPTION
**Pull request by efd6**
## Proposed commit message

See title.

## Checklist

- [x] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Author's Checklist

- [ ]

## Related issues

- For elastic/integrations#8185

### CODE REVIEW

File: CHANGELOG.next.asciidoc [hunk 1]
```diff
@@ -194,6 +194,8 @@ Setting environmental variable ELASTIC_NETINFO:false in Elastic Agent pod will d
 
 *Packetbeat*
 
+- Add metrics for TCP flags. {issue}36992[36992] {pull}36975[36975]
```
Comment (andrewkroh) [inline CHANGELOG.next.asciidoc:197]: Is this changelog line intentional?

**Review by andrewkroh — COMMENTED** [review]
This looks good to me, but I think we need some accompanying documentation additions to explain the purpose of the pipelines and why you would want to ensure they are loaded. We'll need some docs to explain how to load the pipelines manually for cases where a non-Elasticsearch output is used. And lastly the docs will need to reflect the additional privilege that is needed to install the pipelines (`manage_pipeline`). This can be a separate PR.

File: CHANGELOG.next.asciidoc (hunk 1 above)
Comment (efd6) [inline CHANGELOG.next.asciidoc:197]:
```suggestion
```

Comment (efd6) [reply CHANGELOG.next.asciidoc]: No it is not. Removed.

**Review by andrewkroh — APPROVED** [review]

**Pull request merged**